cmd/releasego/wait-latest-mar-go-version.go

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"log"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/microsoft/go-infra/executil"
+	"github.com/microsoft/go-infra/goversion"
+	"github.com/microsoft/go-infra/subcmd"
+)
+
+const marRepo = "mcr.microsoft.com/oss/go/microsoft/golang"
+
+func init() {
+	subcommands = append(subcommands, subcmd.Option{
+		Name:    "wait-latest-mar-go-version",
+		Summary: "Wait (poll) for the latest Microsoft Go images on MAR to match specified versions.",
+		Description: `
+
+Given a list of Go versions, constructs tag names for each one's major version (1.20.3-1 -> 1.20)
+and repeatedly tries to "docker pull" that tag and then probe the image using
+"docker run ... go version" to see if it contains the expected major+minor+patch Go version.
+`,
+		Handle: waitMarGoVersion,
+	})
+}
+
+func waitMarGoVersion(p subcmd.ParseFunc) error {
+	versionList := flag.String(
+		"versions", "",
+		"[Required] A list of full or partial microsoft/go version numbers (major.minor.patch[-revision[-suffix]]). Separated by commas.")
+
+	timeout := flag.Duration("timeout", time.Minute*5, "Time to wait before giving up.")
```

Comment:
I think instead of time.Minute*5 , 5*time.Minute is better approach,[ also how stdlib of Go does this ](https://cs.opensource.google/go/go/+/refs/tags/go1.22.6:src/time/time.go;l=637)

Reply:
Agreed, I think it reads better, more like "5 minutes".